microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -381,21 +381,19 @@ The family group organizer may stop the data collection from the family safety p
 
 
 <!-- ====================================================================== -->
-## Find on Page 
+## Find on page 
 
-Find on Page allows you to search a web page for important keywords. Open **Settings and more (...)** > **Find on Page**. A search box will appear. Type any keyword or phrase into this search box and the page will highlight all locations on the page where your exact search term appears. You can then move through these terms with the **Previous result** and **Next result** buttons. 
+Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.
 
-To help you find exactly what you're looking for, Microsoft Edge uses a Microsoft cloud service to find additional related matches for your search.  This only happens when the **Include related matches** toggle in **Find on Page** is turned on.
+By default, the **Find on page** feature sends data to Microsoft to help find other related matches, including synonyms and alternate spellings.
```

Comment:
```suggestion
By default, the **Include related matches in Find on page** feature allows **Find on page** to find other related matches, including synonyms and alternate spellings. 

When the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms that are sent to Microsoft are deleted immediately after processing occurs.
```
"When" instead of "if" because it's on by default. Didn't include "no data is stored for any period of time" to remove absolute language.